src/System Application/App/Feature Key/src/FeatureManagementImpl.Codeunit.al

```diff
@@ -106,8 +107,10 @@ codeunit 2610 "Feature Management Impl."
             end;
         // If the table extension is not in sync during upgrade then Get() always returns False,
         // so the following insert will fail if the record does exist.
-        if AllowInsert then
+        if AllowInsert then begin
             if FeatureDataUpdateStatus.Insert() then;
```

Comment:
Should this be log inside the if condition?

Reply:
I think we should remove ; and log telemetry only if the insert is successful.

Reply:
This is interface and we need to use interface for Feature Management, so will keep it.